Userland/Libraries/LibGUI/EmojiInputDialog.cpp

```diff
@@ -65,6 +65,7 @@ EmojiInputDialog::EmojiInputDialog(Window* parent_window)
                 auto emoji_text = builder.to_string();
                 auto& button = horizontal_container.add<Button>(emoji_text);
                 button.set_min_size(16, 16);
+                button.set_max_size(16, 16);
```

Comment:
We have `set_fixed_size()` as a shorthand for `set_min_size()` + `set_max_size()`. :^)

Reply:
How stupid of me to miss that, in my excuse I am currently running a fever :^)

Fixed!